WinUtils.cpp

```diff
@@ -14,7 +14,7 @@ namespace commonItems
 std::set<std::string> GetAllFilesInFolderRecursive(const std::string& path)
 {
 	auto validatedPath = path;
-	if (!validatedPath.empty() && (validatedPath[validatedPath.size() - 1] == '/' || validatedPath[validatedPath.size() - 1] == '\\'))
+	if (!validatedPath.empty() && (validatedPath.ends_with('/') || validatedPath.ends_with('\\')))
```

Comment:
Do a quick check that ends_with returns false on an empty string (I'd expect it to). If so, remove the empty() check here.

Reply:
Yeah, there should be no problem.
![image](https://user-images.githubusercontent.com/29546927/99185510-a12bd680-274a-11eb-8b06-1cb826f6902a.png)